jsons/parameters.json

```diff
@@ -299,6 +286,19 @@
         "value_translations": {},
         "value_type": "int"
     },
+    {
+        "default_value": 8,
+        "description": "Factor that is used to multiply the median delta between MS runs in order to define a time threshold, which is then used to separate batches. A measurement duration is allowed to exceed median duration (m) before calling it an interruption. This param defines the factor of m, which if exceeded will define a measurement interruption and therefore allow for batch separation. Default value of 8 is based on -> usually m = 2 to 2.5 min, so this is at least 15 min which is less than the minimum time it takes to clean the ion source",
+        "key_translations": {
+            "assign_analysis_batch_by_runtime_style_1": "number_of_medians",
+            "ursgal_style_1": "batch_seperation_time_factor"
+        },
+        "name": "batch_seperation_time_factor",
+        "tag": [],
+        "triggers_rerun": true,
+        "value_translations": {},
+        "value_type": "int"
+    },
```

Comment:
Interesting, that you had to move it around... I thought wrang sorting would be caught by GA? we would not have merged the PR, if it was failing though...

*Update*: just realised that there is indeed a warning, but not error. We might need to find tune it here!

In my new PR, I have renamed this param as I introduced a typo in it... we should align on that one or first merge yours and then I update the sorting and naming in mine ;)